docs/features.md

```diff
@@ -40,3 +40,4 @@ QMK has a staggering number of features for building your keyboard. It can take
 * [Unicode](feature_unicode.md) - Unicode input support.
 * [Userspace](feature_userspace.md) - Share code between different keymaps and keyboards.
 * [Velocikey](feature_velocikey.md) - Allows changes in RGB animation speed based on WPM/Typing speed.
+* [Raw HID](feature_rawhid.md) - Allows 32-byte bi-directional packet between keyboard and PC Host using HID interface.
```

Comment:
```suggestion
* [Raw HID](feature_rawhid.md) - Allows bi-directional communication between QMK and the host.
```
The docs page also needs to be added to `_summary.md` so it appears in the sidebar.

Reply:
Ok I'll add it to `_summary.md`